### PR TITLE
FSF-5958 | Large cone identification

### PR DESCRIPTION
**Pull request by Flopes55**
Large cone identification was added inside height validation and cylinder validation so perception can identify and differentiate large cones with no added complexity. The testing of this functions was altered to accommodate the changes made.  More information on the development tab for perception in Notion.

### CODE REVIEW

**Comment by yvesb04** [comment]
is foi fechado e ainda n levou merge? precisava disto para deteção de meta

**Comment by Davide64-dev** [comment]
O PR vai ser alterado para a target branch ser a dev e não a main

**Review by yvesb04 — CHANGES_REQUESTED** [review]

File: src/perception/launch/vehicle.launch.py [hunk 1]
```diff
@@ -96,52 +95,53 @@ def generate_launch_description():
...
             DeclareLaunchArgument(
                 "max_height",
                 description="Maximum height of a cluster to be considered a cone",
-                default_value="0.4"
+                default_value="0.55",
+            ),
+            DeclareLaunchArgument(
+                "small_max_height",
+                description="Maximum height of a cluster to be considered a small cone",
+                default_value="0.4",
```
[the remark this answers is not in the document]
Comment (Flopes55) [reply src/perception/launch/vehicle.launch.py:108]: max_height was at 0.4m before and a small cone's height is 0.325m so I kept the small cone height and added a treshold for large cones too (their height is 0.505m), this values still need testing using the recently recorded rosbags with large cones in them.

**Comment by Davide64-dev** [comment]
@Flopes55 overall good just validate this please

**Review by Davide64-dev — APPROVED** [review]

**Review by BrunoDias201907828 — APPROVED** [review]

**Pull request merged**